react/features/base/connection/actions.native.js

```diff
@@ -18,9 +18,12 @@ import {
 /**
  * Opens new connection.
  *
+ * @param {string} [username] - The XMPP user id eg. user@server.com.
```

Comment:
why not call it `jid` or `id`? `username` suggests the domain part is not included.